static/js/StaticPages.jsx

```diff
@@ -3126,6 +3129,330 @@ const JobsPage = memo(() => {
+                        type: {
+                            en: productsData.attributes?.type,
+                            he: productsData.attributes?.localizations?.data[0]?.attributes?.type,
+                        },
+                        url: productsData.attributes?.url,
+                        desc:
+                        {
+                            en: productsData.attributes?.description,
+                            he: heLocalization?.description
+                        },
+                        rectanglion: {
+                            url: productsData.attributes?.rectanglion?.data?.attributes?.url,
+                        },
+                        ctaLabels: ctaLabelsLocalized,
+
+                    };
+                }, {});
+                setProducts(productsFromStrapi);   
+            } catch (error) {
+                console.error("Fetch error:", error);
+                setError("Error: Sefaria's CMS cannot be reached");
+            }
+        } else {
+            setError("Error: Sefaria's CMS cannot be reached");
+        }
+    };
+    
+
+    useEffect(() => {
+        loadProducts();
+    }, []);
+
+    console.log("products: ", products)
+
+    // Generalized function for catching products page analytics
+    const productsAnalytics = (rank, product_cta, label, event) => {
+        gtag("event", `${product_cta}_${event}`, {
+          panel_type: "strapi-static",
+          panel_number: 0,   
+          panel_name: "Products Page",
+          position: rank,
+          experiment: 1,
+          feature_name: label,
+          engagement_type: "static",
+          engagement_value: 0,
+        });
+    }
+
+
+    // The static content on the page inviting users to browse our "powered-by" products
+    const DevBox = () => {
+        return (
+          <div className='productsDevBox'>
+            <p className='productsDevHeader'>
+                <span className="int-en">Powered by Sefaria</span>
+                <span className="int-he">פרויקטים מכח ספריא</span>
+            </p>
+            <p>
+                <span className="int-en">
+                Check out the products our software developer friends from around the world have been building for you! <a href="www.example.com">Explore</a>
+                </span>
+                <span className="int-he">נסו את המוצרים שמפתחי תוכנה וידידי ספריא מרחבי העולם בנו עבורכם! <a href="www.example.com">גלו את הפרויקטים</a></span>
+                
+            </p>
+          </div>
+        );
+      };
+
+    /**
+     * The following are the building block components of an individual product. 
+     */
+     
+    // The title and gray background label for each product
+    const ProductTitle = ({product}) => {
+        return (
+            <div className='productsTitleAndLabel'>
+            <span className="productsTitle">
+                <span className="int-en">{product.titles.en}</span>
```

Comment:
Here and in general, why are we not using InterfaceText and its associated compnents instead of these int-en/he divs? 
IIRC that should work for all React code, and the method used here was only left for actually static html templating?

Reply:
My bad, I got confused about our practices here. I'll fix it.